### ISSUE TITLE
feat(model/v41): assemble full V4.1 text forward

### ISSUE THREAD

**Issue by anthony-chaudhary** [anthony-chaudhary/fak#12901]
# feat(model/v41): assemble full V4.1 text forward

```routing
lane: model
paths: ["internal/model/v41_forward.go", "internal/model/v41_forward_test.go"]
expected_steps: 7
repo: fak
public_issue: anthony-chaudhary/fak#12640
```

## Parent context
Parent: #12640 (native DeepSeek V4.1 Flash execution and physical qualification).

## Current state
Implementing this leaf's through-line is not yet started; the parent track is open.

## Problem frame
- Centrality: Core ? the terminal step that makes native generation testable.
- P1: advanced - the terminal step that makes native generation testable.
- P2: advanced - connects all leaves behind Prefill/Step.
- P3: preserved - other model families unchanged.
- P4: advanced - missing stage fails closed.

## Why now
This leaf is on the critical path for native V4.1 execution and its parent track is open.

## Core through-line
Connect the V4.1 leaves behind `Session.Prefill`/`Session.Step`: embedding, 40 transformer layers
(mHC + CED/CSA2 + Router + Engram + MoE), final norm, and head logits, against the pinned
reference at revision `dba1be0a40aa45a94ad051997016db3960a90277`. Hard fail-closed on any missing
stage.

## Working spine
Intake -> bounded change -> focused witness -> landed receipt.

## Gold-plating boundary
No GPU qualification, no GGUF loading, no streaming, no vision or DSpark. Text forward assembly
only, exercised by the independent oracle fixture. This leaf does not by itself qualify the full
official checkpoint for generation.

## Witness
```bash
go -C ..\fak test ./internal/model -run 'TestV41Forward' -count=1
```

### File:Line Seam
`internal/model/v41_config.go` admission -> `v41_forward.go` assembly

### Blast Radius & Affected Lanes
- **Primary Lane:** `model`
- **Unaffected Lanes:** all other model families unchanged

### Quarantined Fallback Mechanism
Any unimplemented stage returns a typed error; the official checkpoint never routes to a generic
forward.

## Scoped Acceptance Criteria
- [ ] [SW-VERIFIED] Reduced V4.1 model produces logits matching the independent oracle.
- [ ] [SW-VERIFIED] Prefill and Step produce consistent continuation on a two-token case.
- [ ] [SW-VERIFIED] A missing stage fails closed with a typed error.
- [ ] [SW-VERIFIED] Focused tests and `go vet ./internal/model` pass.

## Done condition
The scoped acceptance criteria above pass with the cited witness; no sibling lane regresses.

## Acceptance gate
Focused witness passes, package tests are green, vet and 5-gate boundary checks are clean.

## Work estimate
Estimate: 2 points (bounded leaf).

## Overall completion contribution
Contribution: 2/40 points (provisional baseline).

## Completion standard
development

## Closure binding
The resolving commit cites this issue with Fixes/Ref and carries the repository-qualified lane trailer.

**Comment by anthony-chaudhary** [comment]
Resolved in commit 7862236e1 (origin/main). Verification witness: `go test ./internal/model -run TestV41Forward -count=1` -> ok; full `./internal/model` package green; gofmt + vet clean. This lands the text-forward assembly seam (Session.Prefill/Step + Model.Forward branches) that unblocks the dependent oracle/#12908 and the physical qualification track #12640. Note: no physical checkpoint execution is claimed here.
